Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
+RUN npm i kosmtik-overpass-layer
+RUN npm i kosmtik-fetch-remote
+RUN npm i kosmtik-overlay
+RUN npm i kosmtik-open-in-josm 
+RUN npm i kosmtik-map-compare 
+RUN npm i kosmtik-osm-data-overlay 
+RUN npm i kosmtik-mapnik-reference
+RUN npm i kosmtik-geojson-overlay
+
+RUN echo "plugins:" >> /tmp/.kosmtik-config.yml
```

Comment:
Here applies the same about layers. You could either sacrifice readability a bit and move everything you have to echo in one string with `\n` or you could also group commands together with && thus writing:
RUN echo "plugins:" >> /tmp/.kosmtik-config.yml && echo " ..." ...